modules/openapi-generator/src/main/resources/dart2/enum.mustache

```diff
@@ -78,4 +78,4 @@ class {{{classname}}}TypeTransformer {
 
   /// Singleton [{{{classname}}}TypeTransformer] instance.
   static {{{classname}}}TypeTransformer _instance;
-}
\ No newline at end of file
+}
```

Comment:
Remove the newline

Reply:
It's all good, this is autogenerated code, shouldn't even be committed to version control, there are plenty of generators that leave +5 empty whitespaces at end of files.

Reply:
These new lines were added by AS, but I'm with @agilob on this one.. I always apply dartfmt on the generated client so things like empty lines are removed.. but when we add the effective dart guide the generated code will need to be properly formatted when generated isn't?

Reply:
The problem is that it will change again for the next PR and the next and the next. That is lots of noise that doesn't actually affect the commit. And both enum templates are actually inlined so you suddenly end up with duplicate/triple empty lines between classes. But whatever, up to you.

Reply:
would be nice to properly format the **mustaches** templates and leave the dart fonts reformat for the user.. I find very confusing to read if else conditions on the current mustache templates